raster/r.report/header.c

```diff
@@ -1,6 +1,7 @@
 #include <string.h>
 #include <grass/raster.h>
 #include "global.h"
+#include <grass/parson.h>
```

Comment:
@cwhite911 please also submit this file (will be locally in `include/grass/parson.h`).